subsys/bluetooth/controller/ll_sw/lll_sync.h

```diff
@@ -4,6 +4,12 @@
  * SPDX-License-Identifier: Apache-2.0
  */
 
+
+struct lll_sync_iso {
+	struct lll_hdr hdr;
+	uint8_t is_enabled:1;
+};
+
```

Comment:
Move this to a new file `lll_sync_iso.h`

Reply:
Done